samples/Tutorial/Set.fs

```diff
@@ -47,7 +47,7 @@ type Service internal (decider : Equinox.Decider<Events.Event, Fold.State>) =
         decider.Query id
 
 let create resolve setId =
-    Service(streamName setId |> resolve)
+    Service(target setId |> resolve)
```

Comment:
Could be `Service(target >> resolve)` for consistency

Reply:
in this sample, the Service targets a well-known SetId (the Service ctor takes a `Decider`, not an `'a -> Decider`